anomalib/models/reverse_distillation/lightning_model.py

```diff
@@ -0,0 +1,133 @@
+"""Anomaly Detection via Reverse Distillation from One-Class Embedding.
+
+https://arxiv.org/abs/2201.10703v2
+"""
+
+# Copyright (C) 2022 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions
+# and limitations under the License.
+
+import logging
+from typing import Dict, List, Tuple, Union
+
+from omegaconf import DictConfig, ListConfig
+from pytorch_lightning.callbacks import EarlyStopping
+from pytorch_lightning.utilities.cli import MODEL_REGISTRY
+from torch import Tensor, optim
+
+from anomalib.models.components import AnomalyModule
+
+from .loss import ReverseDistillationLoss
+from .torch_model import ReverseDistillationModel
+
+logger = logging.getLogger(__name__)
+
+
+@MODEL_REGISTRY
+class ReverseDistillation(AnomalyModule):
+    """PL Lightning Module for Reverse Distillation Algorithm.
+
+    Args:
+        input_size (Tuple[int, int]): Size of model input
+        backbone (str): Backbone of CNN network
+        layers (List[str]): Layers to extract features from the backbone CNN
+    """
+
+    def __init__(self, input_size: Tuple[int, int], backbone: str, layers: List[str]):
+        super().__init__()
+        logger.info("Initializing Reverse Distillation Lightning model.")
```

Comment:
Just an idea, but maybe we can move this to the base `AnomalyModule` class, something like
```
logger.info(f"Initializing {self.__class__.__name__} model.")
```

This way we don't need to add this line to every model. 
I guess we could address this separately.